content/sensu-go/5.20/reference/entities.md

```diff
@@ -1127,3 +1138,4 @@ spec:
 [26]: #processes-attributes
 [27]: ../agent/#discover-processes
 [28]: http://man7.org/linux/man-pages/man1/top.1.html
+[29]: ../license/#view-entity-count-and-entity-limit
```

Comment:
```suggestion
[29]: ../license/#view-entity-count-and-entity-limit
[30]: https://stedolan.github.io/jq/
```

Reply:
Oh I'm not sure I would link `jq` here in our docs.